cli/tsc.rs

```diff
@@ -617,6 +617,7 @@ impl TsCompiler {
       "esModuleInterop": true,
       "incremental": true,
       "inlineSourceMap": true,
+      "isolatedModules": unstable,
```

Comment:
I wonder if there is a way to make this clearer what is intended without running afoul of clippy.  I know it is an extra line but maybe a `let isolated_modules = unstable;` with a comment with a TODO to ensure it gets changed for 1.5.0 (I know we are unlikely to forget, but still...)